pkg/agent/cniserver/interface_configuration_windows.go

```diff
@@ -42,7 +42,7 @@ type ifConfigurator struct {
 	epCache    *sync.Map
 }
 
-func newInterfaceConfigurator(ovsDataPathType string) (*ifConfigurator, error) {
+func newInterfaceConfigurator(ovsDataPathType string, isOvsHardwareOffload bool) (*ifConfigurator, error) {
```

Comment:
@moshe010 do you mind renaming the remaining `isOvsHardwareOffload` to `isOvsHardwareOffloadEnabled`?

Reply:
done

Reply:
did you change something? I still see the same set of occurrences throughout the code.

Reply:
yes I force push it see https://github.com/vmware-tanzu/antrea/compare/f4d50e8a331f89df56a8e6e1a5e8d6258b85c19d..6cf7886a8bdbe992641c070f1159db160398e7b8

Reply:
the commit seems to change one occurrence only, I counted about half a dozen throughout the diff.

Reply:
sorry I will fix that